code-snippets/nested_dynamics.hs

```diff
@@ -1,14 +1,14 @@
+-- A slightly contrived example just to demonstrate use of nested dynamics
```

Comment:
This file has been added to reflex-examples repo https://github.com/reflex-frp/reflex-examples/pull/17
So it will not be a part of docs here.